geomesa-convert/geomesa-convert-json/src/main/scala/org/locationtech/geomesa/convert/json/JsonConverterFactory.scala

```diff
@@ -42,6 +42,7 @@ class JsonConverterFactory extends AbstractConverterFactory[JsonConverter, JsonC
       val reader = new JsonReader(new InputStreamReader(is, StandardCharsets.UTF_8))
       reader.setLenient(true)
 
+      // TODO: We shouldn't use deprecated methods
```

Comment:
The instantiation of `JsonParser` and the call `parser.parse` are deprecated.

Will replacing these with newer, current methods cause any backwards compatibility issues?

Reply:
hmm not sure... we could probably change it for the 5.0 release